internal/matcher/controller.go

```diff
@@ -45,6 +45,11 @@ func (mc *Controller) Match(ctx context.Context, records []*claircore.IndexRecor
 		return map[string][]*claircore.Vulnerability{}, nil
 	}
 
+	remoteMatcher, matchedVulns, err := mc.queryRemoteMatcher(ctx, interested)
```

Comment:
Let's derive a ctx here with some safe deadline and pass it to the mc.queryRemoteMatcher method.

Reply:
@arajkumar - since these are HTTP requests, lets bump it up to 60s for now.

Reply:
Sure, just fixed it. PTAL.